tests/e2e/scripts/stablePool.json

```diff
@@ -0,0 +1,7 @@
+{
+	"initial-deposit": "1000000stake,1000uosmo",
```

Comment:
nit: I think it should be all one TAB back. Correct me if I am wrong please

Reply:
I saw that every json file in scripts folder is the same as this file, so i think it's no need to all TAB back

Reply:
got it, it just seems like there is double tab on every line